cbits/CastFloatWord.cmm

```diff
@@ -0,0 +1,76 @@
+#include "Cmm.h"
```

Comment:
Same for this file. It would be good to have a reference where the original code comes from

Reply:
Cmm.h is part of your Haskell installation :) `./lib/ghc-8.8.3/include/Cmm.h` - I can make a note to that effect.

Reply:
I understand that it comes with ghc, but it doesn't tell us who actually wrote the piece of code. Also who wants to installing ghc in order to look at the original.

It is better to just add a link, which will let everyone know where the code came from, and in the future allow us to see quickly if it has changed or not. https://gitlab.haskell.org/ghc/ghc/-/blob/6d172e63f3dd3590b0a57371efb8f924f1fcdf05/libraries/base/cbits/CastFloatWord.cmm

Reply:
Done